MiniApp/Progress+UIViewController.swift

```diff
@@ -20,11 +20,11 @@ extension UIViewController {
         })
     }
 
-    func displayErrorAlert(title: String, message: String, dismissController: Bool) {
+    func displayAlert(title: String, message: String, dismissController: Bool) {
         DispatchQueue.main.asyncAfter(deadline: .now()) {
             self.dismiss(animated: true, completion: {
                 let alert = UIAlertController(title: title, message: message, preferredStyle: .alert)
-                alert.addAction(UIAlertAction(title: NSLocalizedString("OK", comment: ""), style: .cancel, handler: { (_: UIAlertAction!) in
+                alert.addAction(UIAlertAction(title: NSLocalizedString("ok", comment: ""), style: .cancel, handler: { (_: UIAlertAction!) in
```

Comment:
Maybe an extension that just lets you do `"ok".localized` would be better.

Reply:
well, I could, but I think you can't use automatized string extraction anymore (`xcrun extractLocStrings *.swift`)